### ISSUE TITLE
False warning from prefer-invoke-map

### ISSUE THREAD

**Issue by mik01aj** [wix/eslint-plugin-lodash#93]
On this code:
```
_.map(displayedGenres, ({slug}) => {
    return parallel(slug);
})
```
I get this warning:
```
  28:16  warning  Prefer _.invokeMap over map to a method call  lodash/prefer-invoke-map
```
But this is wrong because AFAIK I can't refactor this to [`_.invokeMap`](https://lodash.com/docs#invokeMap).

**Comment by ganimomer** [comment]
This is definitely a bug. On it.

**Comment by ganimomer** [comment]
Fixed in version `1.10.1`.

**Comment by mik01aj** [comment]
thanks! 🎉  🙌
